bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCflamingos2.html

```diff
@@ -121,81 +120,86 @@
                 <td colspan="3"><b>Spectral distribution:</b> <i><span>(<a href="#" onclick="window.open('http://www.gemini.edu/?q=node/10257#spectral','mywindow','width=1000,height=700,toolbar=no,location=no,directories=no,sta tus=no,menubar=no,scrollbars=yes,copyhistory=no,resizable=no');return false">more info</a>)</span></i><br />
                     <i>Choose one SED, the redshift and extinction</i></td>
             </tr>
-            <tr>
 
+            <tr>
                 <td><input value="LIBRARY_NON_STAR" name="Distribution" type="radio" /></td>
-                <td colspan="2">Library spectrum of a non-stellar object <select name="nsSpectrumType" size="1">
-                    <option value="elliptical-galaxy">
-                        elliptical galaxy</option>
-                    <option value="spiral-galaxy" selected="selected">
-                        spiral galaxy (Sc)</option>
-                    <!--<option value="QSO">QSO</option>-->
-                    <option value="QSO">QSO (80-855nm)</option>
-                    <option value="QSO2">QSO (276-3520nm)</option>
-
-                    <option value="Orion-nebula">
-                        HII region (Orion)</option>
-                    <option value="Planetary-nebula">Planetary nebula (NGC7009: 100-1100nm)</option>
-                    <option value="Planetary-nebula2">Planetary nebula (IC5117: 480-2500nm)</option>
-
-                </select></td>
+                <td colspan="2">Library spectrum of a non-stellar object
+                    <select name="nsSpectrumType" size="1">
+                        <option value="Mars">Mars (540 nm - 5.0 µm)</option>
+                        <option value="Jupiter">Jupiter (530 nm - 5.9 µm)</option>
+                        <option value="Saturn">Saturn (530 nm - 5.9 µm)</option>
+                        <option value="Uranus">Uranus (540 nm - 5.0 µm)</option>
+                        <option value="Neptune">Neptune (540 nm - 5.0 µm)</option>
+                        <option value="none" disabled></option>
```

Comment:
It seems there are a lot of new options here? My question is if this will necessitate changes in the ITC servlet to handle the new values.

Reply:
This is where I put the new solar system SEDs (Mars, Jupiter, Saturn, Uranus, and Neptune).  The servlet will look for the SED files that have those names with the ".nm" extension.